core/src/main/scala/cats/data/Nested.scala

```diff
@@ -167,6 +167,11 @@ private[data] sealed abstract class NestedInstances12 {
     new NestedInvariant[F, G] {
       val FG: Invariant[λ[α => F[G[α]]]] = Invariant[F].composeContravariant[G]
     }
+
+  implicit def catsDataDistributiveForNested[F[_]: Distributive, G[_]: Distributive]: Distributive[Nested[F, G, ?]] =
```

Comment:
It needs to go above the `Functor` instance

Reply:
ok.